### PR TITLE
bugfix #10: WIN32: simple_uart_read not return number of read bytes

### PR DESCRIPTION
**Pull request by akaeba**
bugfix for #10

### CODE REVIEW

File: simple_uart.c [hunk 1]
```diff
@@ -82,8 +82,11 @@ int simple_uart_read(struct simple_uart *sc, void *buffer, int max_len)
         SetCommTimeouts(sc->port, &commTimeout);
     }
 
-    if (!ReadFile (sc->port, buffer, max_len, (LPDWORD)&r, NULL) != 0)
+    Sleep(50);
```
Comment (AndreRenaud) [inline simple_uart.c:85]: I'm not sure about this `Sleep(50);` - is that necessary? That will put a pretty hard rate-limit on the serial port, which doesn't seem ideal.

Comment (akaeba) [reply simple_uart.c:85]: Hi André, the Problem on My Site was, that i Misses sporadically Bytes. This wait solved ist for me. But ist should also possible to solve ist on My application

Comment (AndreRenaud) [reply simple_uart.c:85]: If possible, I'd rather that this wasn't in `simple_uart.c` - some applications will want to fail early if no data is immediately available.

Comment (akaeba) [reply simple_uart.c:85]: I Van adjust the pr

Comment (AndreRenaud) [reply simple_uart.c:85]: Yes, could you drop this line from the PR? Thanks

Comment (akaeba) [reply simple_uart.c]: removed

**Pull request merged**